packages/spec-api/src/types.ts

```diff
@@ -80,6 +87,7 @@ export interface ServiceRequest {
    */
   params?: Record<string, unknown>;
   headers?: Record<string, unknown>;
+  files?: ServiceRequestFile[];
```

Comment:
I considered the choice between `Multer.File[]` vs `ServiceRequestFile[]`. Here, the Multer.File has several properties (such as stream, filename, sourcePath, destinationPath, size, encoding, etc.) which are not really needed but is mandatory. This seemed unnecessary and ServiceRequestFile approach seemed much simpler and cleaner.

Reply:
yeah it is better to use our own type., Any library that we use behind the scene is an implementation detail that shouldn't get exposed